smoke-test/tests/consistency_utils.py

```diff
@@ -27,15 +27,13 @@ def infer_kafka_broker_container() -> str:
     return lines[0]
 
 
-KAFKA_BROKER_CONTAINER: str = str(
```

Comment:
This should likely be converted to a test fixture. Resolve it once, lazy or not, but then inject it into the tests like we did for the graph_client and auth_session. This way it would not only be resolved when needed but then only be the tests which require it.

Reply:
If we make it a test fixture then it will still try to resolve it. That is not needed for the cases where `USE_STATIC_SLEEP` = `true`